client/lib/src/process_event.rs

```diff
@@ -19,38 +17,71 @@ pub fn process_event<UpdateServer>(
   view: &mut view::T,
   client: &client::T,
   event: Event,
+  last_key: &mut Option<VirtualKeyCode>,
 ) where UpdateServer: FnMut(protocol::ClientToServer)
 {
   match event {
-    Event::KeyDown{keycode, repeat, ..} => {
-      keycode.map(|keycode| {
-        if !repeat {
-          key_press(update_server, view, client, keycode);
+    Event::WindowEvent { event: WindowEvent::CursorMoved { .. }, .. } => {}
+    Event::WindowEvent { event: WindowEvent::MouseWheel { .. }, .. } => {}
+    Event::WindowEvent { event: WindowEvent::AxisMotion { .. }, .. } => {}
+    Event::DeviceEvent { event: DeviceEvent::Motion { .. }, .. } => {}
+    Event::DeviceEvent { event: DeviceEvent::MouseMotion { .. }, .. } => {}
+    _ => warn!("event: {:?}", event),
```

Comment:
This warns on keyboard input. What's the thinking behind warning on any events that aren't the ones we handle? It seems fine for there to be other events?